docs/faq.md

```diff
@@ -68,7 +72,7 @@ No. The channel behaves like a regular channel.
 
 ### What happens if the seller force closes a channel before the maturity date?
 
-If the lender force closes they will be banned from the market. This will not be possible in the future.
+Script level enforcement (WIP) will lock the maker's funds in the channel for the lease period. By now, we use a feature in `lnd` to prevent channels from being _cooperatively closed_ by the maker until the expiry height \(what we call the `thaw_height`\).
```

Comment:
Does the WIP refer to this sentence or to the script level enforcement in general?

Reply:
Yes, we do not have a way to enforce it yet right?

Reply:
The PR was merged and deployed, but you need to actively opt-in into script-enforced leases (if you're running `lnd v0.14.x`). So it's not WIP anymore, just not the default yet.